src/Volume.js

```diff
@@ -145,18 +137,10 @@ export default class Volume {
       this.pixel_size[2] = values[2];
     }
 
-    var physSizeMin = Math.min(
-      this.pixel_size[0],
-      Math.min(this.pixel_size[1], this.pixel_size[2])
-    );
-    var pixelsMax = Math.max(
-      this.imageInfo.width,
-      Math.max(this.imageInfo.height, this.z)
-    );
-    var sx =
-      ((this.pixel_size[0] / physSizeMin) * this.imageInfo.width) / pixelsMax;
-    var sy =
-      ((this.pixel_size[1] / physSizeMin) * this.imageInfo.height) / pixelsMax;
+    var physSizeMin = Math.min(this.pixel_size[0], Math.min(this.pixel_size[1], this.pixel_size[2]));
+    var pixelsMax = Math.max(this.imageInfo.width, Math.max(this.imageInfo.height, this.z));
+    var sx = ((this.pixel_size[0] / physSizeMin) * this.imageInfo.width) / pixelsMax;
+    var sy = ((this.pixel_size[1] / physSizeMin) * this.imageInfo.height) / pixelsMax;
```

Comment:
aren't you replacing 'vars'?

Reply:
not yet -- I only replaced a few because of eslint errors.  To do all of them will be a separate PR.  Considering going toward typescript too.